### PR TITLE
#11421: Replace import tt_lib to ttnn in model experimental

### PR DESCRIPTION
**Pull request by ruthreshx**
### Ticket
[Link to Github Issue](https://github.com/tenstorrent/tt-metal/issues/11421)

### Problem description
Move ttlib to ttnn

### What's changed
- [x] squeezebert (No occurances)
- [x]  squeezenet (No occurances)
- [x]  ssd
- [x]  stable_diffusion
- [x]  swin
- [x]  synthetic_gradients

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: models/experimental/ssd/tests/perf_ssd.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import torch
 from loguru import logger
-import tt_lib
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/ssd/tests/perf_ssd.py:7]: Remove, ttnn not used in this file

File: models/experimental/ssd/tests/test_ssd_backbone.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
     SSDLite320_MobileNet_V3_Large_Weights,
     ssdlite320_mobilenet_v3_large as pretrained,
 )
-import tt_lib
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/ssd/tests/test_ssd_backbone.py:12]: Same [here](https://github.com/tenstorrent/tt-metal/pull/11553/files#r1721486440)

File: models/experimental/ssd/tt/ssd_mobilenetv3_convlayer.py [hunk 3]
```diff
@@ -6,8 +6,8 @@
 import torch.nn as nn
 import ttnn
 
-import tt_lib
-import tt_lib.fallback_ops as fallback_ops
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/ssd/tt/ssd_mobilenetv3_convlayer.py:9]: importing twice

File: models/experimental/stable_diffusion/tests/perf_batched_stable_diffusion.py [hunk 4]
```diff
@@ -225,7 +225,7 @@ def test_perf(device, expected_inference_time, expected_compile_time):
         # predict the noise residual
         with torch.no_grad():
             tt_noise_pred = tt_unet(tt_latent_model_input, _t, encoder_hidden_states=tt_text_embeddings)
-            ttl.device.Synchronize(device)
+            ttnn.L1_MEMORY_CONFIG(device)
```
Comment (mcw-anasuya) [inline models/experimental/stable_diffusion/tests/perf_batched_stable_diffusion.py:228]: ttnn.synchronize_device(device)

File: models/experimental/stable_diffusion/tt/embeddings.py [hunk 5]
```diff
@@ -9,9 +9,9 @@
 import ttnn
 
 
-import tt_lib as ttl
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/stable_diffusion/tt/embeddings.py:12]: imported twice

**Review by mcw-anasuya — APPROVED** [review]

**Review by boris-drazic — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**